rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# Technical Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
+
+`[Kevin Osborn](mailto:kosborn2@ucsc.edu)`
+
+`[Jodi Hirschman](mailto:jhirsch@broadinstitute.org)`
+
+`[Matthew Spier](mailto:mspeir@ucsc.edu)`
+
+`[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)`
+
+## Shepherd
+
+***Leave this blank. This role is assigned by DCP PM to guide the Author(s) through the RFC process.*
+
+## Motivation
+
+Beta 2 testing will begin soon and we’ve decided to collect feedback
+through our Zendesk ticketing system. We therefore need to quickly
+determine a process for handling larger numbers of Zendesk tickets than
+we have received to date. Several of us have been thinking about this
+for awhile and drew up an initial SOP that we refined at the last DCP
+F2F (Feb. 2019); we present it here as this RFC. 
+
```

Comment:
Maybe we should break the Motivation section up into 2 paragraphs. One that highlights the immediate need (i.e. Beta2) and one that highlights the longterm sustainability need (i.e. handling all user queries post-Beta2).

Reply:
I've expanded the Motivation section.